### PR TITLE
Delete HOS headset from warden's locker

### PR DESCRIPTION
**Pull request by kosticia**
## About the PR
Removes Hos headset from warden locker

## Why / Balance
Warden isn't command staff 

## Technical details
Watch the code

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- delete: Removed HOS headset from warden locker

### CODE REVIEW

**Comment by Emisse** [comment]
looks good just need you to update the copyright field of the headsets meta.json at the top

**Review by Emisse — APPROVED** [review]

**Comment by kosticia** [comment]
> looks good just need you to update the copyright field of the headsets meta.json at the top

Done

File: Resources/Prototypes/Entities/Clothing/Ears/headsets.yml [hunk 1]
```diff
@@ -288,3 +288,21 @@
       sprite: Clothing/Ears/Headsets/freelance.rsi
     - type: Clothing
       sprite: Clothing/Ears/Headsets/freelance.rsi
+
+- type: entity
+  parent: [ClothingHeadset, BaseCommandContraband]
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Clothing/Ears/headsets.yml:293]:
```suggestion
  parent: [ClothingHeadset, BaseRestrictedContraband]
```
The warden is not command, so it being command restriced shows up as contraband for them
grafik
For the same reason they should probably not have a command encryption key either.

File: Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml [hunk 2]
```diff
@@ -155,3 +155,20 @@
       sprite: Clothing/Ears/Headsets/freelance.rsi
     - type: Clothing
       sprite: Clothing/Ears/Headsets/freelance.rsi
+
+- type: entity
+  parent: [ClothingHeadsetAlt, BaseCommandContraband]
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml:160]:
```suggestion
  parent: [ClothingHeadsetAlt, BaseRestrictedContraband]
```

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
Thanks for fixing it. I opened a maintainer discussion about if the warden is supposed to have a command encryption key. I will come back to you once we have a conclusion.

**Comment by slarticodefast** [comment]
Hey, we had a maintainer vote and came to the conclusion that the warden should not have a command encryption key since they are not command staff themselves. As such they no longer need a special headset, but can be given a normal security headset instead. Could you adjust your PR for that?

**Comment by kosticia** [comment]
> Hey, we had a maintainer vote and came to the conclusion that the warden should not have a command encryption key since they are not command staff themselves. As such they no longer need a special headset, but can be given a normal security headset instead. Could you adjust your PR for that?

Done.

**Comment by kosticia** [comment]
All these likes... :trollface:

**Review by slarticodefast — APPROVED** [review]
Yeah, sorry that your sprite did not make it.
But the warden should not have a command headset in the first place.
Anyways, thank you for your contribution.

**Pull request merged**